docs/angle_calculation.md

```diff
@@ -2,12 +2,14 @@
 
 Here we explain the implementation of MOM6 angle calculation in regional-mom6, which is the process by which regional-mom6 calculates the angle of curved horizontal grids (``hgrids``).
 
-**Issue:** On a curved hgrid, we have to rotate the boundary conditions according to the angle the grid is rotated from lat-lon coordinates (true north vs model north). MOM6 calculates the angle internally. ``rotation.py`` copies that MOM6 calculation into python so we can rotate it based on that calculation. The issue is that usually users provide hgrids in MOM6 with an angle field (``angle_dx``) and we were using that field. We don't necessarily trust that the user calculation doesn't have slight differences, so we implemented a way to use MOM6s angle calculation instead. In short, MOM6 doesn't actually use the user-provided ``angle_dx`` field in input hgrids, but internally calculates the angle. 
+**Issue:** On a curved hgrid, we have to rotate the boundary conditions according to the angle the grid is rotated from lat-lon coordinates (true north vs model north). Although horizontal grids supplied by users will contain an `angle_dx` field, MOM6 ignores this field entirely and calculates its own grid angles internally. 
 
-**Solution:** To accomodate this fact, when we rotate our boundary conditions, we implemented MOM6 angle calculation in a file called "rotation.py", and adjusted functions where we regrid the boundary conditions.
+**Solution:** To be consistent with MOM6's treatement of grid angles, when we rotate our boundary conditions, we implemented MOM6 angle calculation in a file called "rotation.py", and included this in the the boundary regridding functions by default.
 
 
-## MOM6 process of angle calculation (T-point only)
+## Boundary rotation algorithm
+Steps 1-5 are the same as MOM6's internal angle calculation. Step 6 is an additional step required to apply this algorithm to the boundaries
```

Comment:
```suggestion
Steps 1-5 replicate the angle calculation as done by MOM6. Step 6 is an additional step required to apply this algorithm to the boundaries.
```

Reply:
Either here, or in the docstring of the method that replicates the MOM6 angle calculation, it would be good to point to the MOM6 source code. I'd say the docstring is more appropriate place to do this, if it's not done already.

Reply:
What does MOM6 do at the boundaries? Why don't we replicate that instead of using Step 6? Then we don't need to differentiate or explain, right?